sdk/typescript/src/signers/txn-data-serializers/txn-data-serializer.ts

```diff
@@ -45,6 +46,7 @@ export interface PayTransaction extends TransactionCommon {
   recipients: SuiAddress[];
   amounts: number[];
   gasPayment?: ObjectId;
+  gasOwner?: SuiAddress;
```

Comment:
Just to confirm, `gasOwner` will not present in `paySui` and `payAllSui` transactions?

Reply:
In rust, `GasData` is a required sibling field of `TransactionKind`. But the checker makes sure the `gasOwner` to be `sender`.

in TS SDK, maybe we don't need `gasOwner`?

Reply:
I will take a look and remove it if they are not needed in TS SDK